3.analysis/scripts/nbconverted/3.t_tests.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python
```

Comment:
Please output the results of the t-test.

The deliverable is a 3 `.tsv` file:

- Column 1: t-test statistic
- Column 2: p-value
- Column 3: Latent feature
- Column 4: comparison (e.g. "Male vs. Female")

This tidy data format is called "long". Please add the output file to this repo

Reply:
sorry, I meant "4 column tsv file"